### DISCUSSION TITLE
内网环境中，前端和后端不再同一台电脑 前端调用后端会报跨域问题 怎么解决？

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#217] [02.提问]
内网环境中，前端和后端不再同一台电脑 前端调用后端会报跨域问题 怎么解决？

**Comment by zuihou** [comment]
![image](https://user-images.githubusercontent.com/8500410/200731875-4e094076-fb0f-4776-a975-93cd2fd97e3c.png)

.env.development 文件中，配置其他ip的后台服务地址， 不会跨域
